src/Sepes.Infrastructure/Model/Automapper/AutoMappingConfigs.cs

```diff
@@ -30,6 +31,8 @@ public AutoMappingConfigs()
 
 
             CreateMap<Study, StudyListItemDto>();
+            CreateMap<StudyListItemDto, StudyListItemResponse>()
+             .ForMember(dest => dest.Id, source => source.MapFrom(x => x.StudyId));

```

Comment:
Would it be difficult to just have ID on both of these classes=

Reply:
Just had to change one field of the query actually